### PR TITLE
add tryton

### PR DESCRIPTION
**Pull request by looooo**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by looooo** [comment]
@conda-forgepython this is ready from my side.

File: recipes/tryton/recipe.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install ./tryton --no-deps -vv
+
+requirements:
+  host:
+    - pip
+    - python ==${{ python_min }}
```
Comment (beckermr) [inline recipes/tryton/recipe.yaml:21]:
```suggestion
    - python ${{ python_min }}.*
```

**Comment by looooo** [comment]
any ideas why it is stuck?

**Comment by beckermr** [comment]
Job is queued. No idea why.

File: recipes/tryton/recipe.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      imports:
+        - tryton
+      pip_check: true
+
+about:
+  license: GPL-3.0-or-later
+  license_file: tryton/COPYRIGHT
+  summary: Tryton is business software, ideal for companies of any size, easy to use, complete and 100% Open Source.
+  homepage: https://github.com/tryton/tryton
+  repository: https://github.com/booya-at/OpenGlider
```
[the remark this answers is not in the document]
Comment (beckermr) [reply recipes/tryton/recipe.yaml]: If it is, we need to adjust the linter.

[the remark this answers is not in the document]
Comment (wolfv) [reply recipes/tryton/recipe.yaml]: This was just about the URL

**Pull request merged**